api/constants/constants.go

```diff
@@ -422,6 +422,20 @@ const (
 	MaxAssumeStartDuration = time.Hour * 24 * 7
 )
 
+const (
+	// MaxHealthCheckInterval is the minimum interval between resource health
+	// checks.
+	MinHealthCheckInterval = 30 * time.Second
+	// MaxHealthCheckInterval is the maximum interval between resource health
+	// checks. Since timeout must be less than interval, this is effectively the
+	// maximum health check timeout as well.
+	MaxHealthCheckInterval = 300 * time.Second
```

Comment:
Any concerns with allowing even greater values here? Use case: single agent supporting a few thousands databases, rarely contacted individually. We would need a really high interval here. 

For example with 1000 databases checked every 15 minutes each gives us a check every 0.9 seconds, which is very often still.